client/app/vps/import/vps.service.js

```diff
@@ -276,10 +290,28 @@ angular.module('managerApp').service('VpsService', [
       return this.getSelectedVps(serviceName).then((vps) => {
         if (!ips) {
           return $q.reject('No ips');
-        } if (vps && vps.name) {
-          return $http.post([aapiRootPath, vps.name, 'ips', 'reverse'].join('/'), ips, { serviceType: 'aapi' })
-            .then((data) => { result = data.data; });
         }
+
+        if (vps && vps.name) {
```

Comment:
```suggestion
        if (_.has(vps, 'name')) {
```

Reply:
@FredericEspiau since this ticket is in quality check I will merge it.

We could address you suggestion in a separate pull request.

Thanks!